firmware/network/border_router/border_router.c

```diff
@@ -36,6 +39,25 @@ int8_t get_wired_iface(void) {
     return -1;
 }
 
+int8_t get_wireless_iface(void) {
+    int max_ifaces = gnrc_netif_numof();
+    if (max_ifaces > 0) {
+        gnrc_netif_t *iface;
+         for (uint8_t i = 0; i < ARRAY_SIZE(radio_devices); i++) {
+            iface = gnrc_netif_get_by_type(radio_devices[i], NETDEV_INDEX_ANY);
+            if (iface != NULL) {
+                break;
+            }
+        }
+        if (iface != NULL) {
+            return iface->pid;
+        } else {
+            return -1;
+        }
+    }
+    return -1;
+}
```

Comment:
This it's the same that uses of radio module get_ieee802154_iface(), you could reuse that function

Reply:
The function get_ieee802154_iface() throws an error, undefined and redefined